python/pyfunc-server/docker/local.Dockerfile

```diff
@@ -12,15 +12,25 @@
 # See the License for the specific language governing permissions and
```

Comment:
this is identical to `python/sdk/merlin/docker/pyfunc.Dockerfile`? can we remove one of them and refer to the other?

Reply:
Removed